app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+    </div>
+</section>
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my pupils",
+          text: "I enjoy showing my pupils that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
+          accreditation: "Rotna Roy, Maths Lead at a school in London",
+        }
+      ],
+      colors: %w[yellow],
+      numbered: false
+    ) %>
+  </section>
+
+<section class="col col-space-m col-720">
+<h2>Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in maths.</p>
+<p>Maths teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
+<p>Your training provider may need you to brush up on your computing knowledge before you become a maths teacher. You can do a <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE) course</a> before you start your teacher training to top up your subject knowledge. </p>
+<p><a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out more about the qualifications needed to teach</a>.</p>
+<p><a href="/non-uk-teachers">Non-UK citizens can find out more about training to teach in England</a>.</p>
+
+<h2>Fund your teacher training</h2>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee maths teachers</a>. </p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get <a href="/funding-and-support">extra funding support</a> if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding.</p>
+<p>You can <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">find out more about funding for non-UK citizens</a>.</p>
+
+<h2>Paid internship</h2>
+<p>If you’re currently doing an undergraduate or master's degree and are interested in teaching computing, you could apply for a <a href="/is-teaching-right-for-me/teaching-internship-providers">paid teaching internship</a>.</p>
+<p>The 3-week programme could help you to understand what it’s really like in the classroom and get a feel for school life.</p>
+<p>You can also find out what it's like to be in a classroom through <a href="/is-teaching-right-for-me/get-school-experience">unpaid school experience</a>.</p>
```

Comment:
@caitlin-reidy we were going to lose the line about Get school experience and focus on paid internships as it's something the candidate support team are keen to promote and this is one of the subjects that qualifies.